library/aos_version.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
```

Comment:
I would suggest an additional role that is included via dependency in meta/main.yml from common, masters, and nodes roles.